feature/experimental/policy/policy_vrf_selection/otg_tests/policy_based_traffic_steering_default_to_nondefault/README.md

```diff
@@ -0,0 +1,311 @@
+| DSCP value AF4, Destination IPv4Prefix10/24"                       | 35                        |
+| DSCP value AF4, Destination IPv6Prefix10/24"                       | 35                        |
+
+
+
+   * DUT has 2 VRFs, Default and Non-Default. Configure Route leaking from the Default to the Non-default VRF to share reachability between VRFs.<br>
+
+
+### Flows:
+
+Flows from ATE1:Port1 to DUT:Port1
+
+   * Following are destinations.
+       * IPv4Prefix1/24 to IPv4Prefix6/24 [DSCP:BE1]
+       * IPv6Prefix1/64 to IPv6Prefix6/64 [DSCP:BE1]
+       * IPv4Prefix2/24 to IPv4Prefix7/24 [DSCP:AF1]
+       * IPv6Prefix2/64 to IPv6Prefix7/64 [DSCP:AF1]
+       * IPv4Prefix3/24 to IPv4Prefix8/24 [DSCP:AF2]
+       * IPv6Prefix3/64 to IPv6Prefix8/64 [DSCP:AF2]
+       * IPv4Prefix4/24 to IPv4Prefix9/24 [DSCP:AF3]
+       * IPv6Prefix4/64 to IPv6Prefix9/64 [DSCP:AF3]
+       * IPv4Prefix5/24 to IPv4Prefix10/24 [DSCP:AF4]
+       * IPv6Prefix5/64 to IPv6Prefix10/64 [DSCP:AF4] br><br><br>
+
+
+### Following Health checks to be run Pre and Post every sub test
+  * No system/kernel/process/component coredumps
+  * No high CPU spike or usage on control or forwarding plane
+  * No high memory utilization or usage on control or forwarding plane
+  * No processes/daemons high CPU/Memory utilization
+  * No generic drop counters
+    * QUEUE drops
+    * Interfaces
+    * VOQ
+    * Fabric drops
+    * ASIC drops
+  * No flow control frames tx/rx
+  * No CRC or Layer 1 errors on interfaces
+  * No config commit errors
+  * No system level alarms
+  * In spec hardware should be in proper state
+    * No hardware errors
+    * Major Alarms
+  * No HW component or SW processes crash
+Openconfig-paths for the above are covered in the `OpenConfig Path and RPC Coverage` section below<br><br><br>
+
+
+
+
+
+**RT-3.33.1: [Baseline] Traffic flow between ATE1:Port1 and ATE2:Port2 via DUT's Default VRF**
+  * Please run all the health checks suggested above prior to starting the test as a baseline.
+  * Start all the Flows from ATE1:Port1 to DUT:Port1 
+  * DUT has the following VRF selection policy
+    * Statement1: traffic matching any IPV4 0r IPV6 destination with any DSCP value, Punt to default vrf <br>
+  * Expectations:
+    * All traffic received in the Default VRF destined for any IPv4Prefix is sent back to ATE2:Port2 based on LPM lookup of EBGP learnt routes. 
+    * Run post test Health checks above to ensure no anomalies<br><br><br>
+
+
+**RT-3.33.2: Identify traffic by DSCP (BE1) and punt to non-default VRF**
+  * Please run all the health checks suggested above prior to starting the test.
+  * Start all the Flows from ATE1:Port1 to DUT:Port1 
+  * DUT has the following VRF selection policy
+    * Statement1: Traffic matching any IPV4 or IPV6 destination with DSCP value BE1, Punt to non-default vrf
+  * Statement2: Traffic matching any IPV4 or IPV6 destination with DSCP value AF1,AF2,AF3,AF4, Punt to default vrf <br>
```

Comment:
Indentation issue -- please check the rendered version of the file to check the bullets as as required (this one needs indenting).